src/fields/models/Boolean.ts

```diff
@@ -106,7 +106,7 @@ export function createDvField(
 ): void {
     attrs.cls = "value-container"
     const checkbox: HTMLInputElement = dv.el("input", "", { ...attrs, "type": "checkbox" })
-    checkbox.checked = p[managedField.name]
+    checkbox.checked = managedField.value
```

Comment:
This change seems simpler to me than my previous approach, is there anything wrong with leveraging the managed field (by passing existing field in to the modifier)?